src/main/resources/templates/js/maps.js

```diff
@@ -0,0 +1,63 @@
+function myMap() {
+    var mapProp= {
+        center:new google.maps.LatLng(39.103119, -84.512016),
+        zoom:10,
```

Comment:
Use this JavaScript file if you end up having multiple pages that use google maps. That way, you can reuse your code to create the map and markers by referring to one file instead of having multiple html each with separate inline JavaScript. Using this file is only necessary in that situation.